pkgs/flox-cli-tests/default.nix

```diff
@@ -141,6 +141,9 @@ writeShellScriptBin PROJECT_NAME ''
   esac
   export PROJECT_TESTS_DIR;
 
+  GIT_LATEST_TAG=$(git describe --tags --abbrev=0)
+  export GIT_LATEST_TAG
+
```

Comment:
This doesn't work for the "remote" jobs in CI. Have asked dev support team for input:

- https://flox-dev.slack.com/archives/C055P1JCP5L/p1737558144962889